java/code/webapp/WEB-INF/pages/activationkeys/create.jsp

```diff
@@ -6,7 +6,7 @@
     <body>
         <rhn:toolbar base="h1" icon="header-activation-key"
                      imgAlt="activation-keys.common.alt"
-                     helpUrl="/docs/reference/systems/activation-keys.html"
```

Comment:
I wonder, would it be possible to use the `docsLocale` variable set in the `layout_head.jsp`?

Reply:
I thought about that one. But I couldn't figure out a way to make that work.

Reply:
@ncounter Do you know if it is possible?

Reply:
I think I missed this comment/question?

Wouldn't it be the same as https://github.com/uyuni-project/uyuni/blob/master/java/code/webapp/WEB-INF/decorators/layout_head.jsp#L66 ?